app/decorators/professional_experience_decorator.rb

```diff
@@ -8,6 +8,17 @@ def start_date
   end
 
   def end_date
-    model.start_date.to_date.to_fs(:date)
+    return I18n.t(:present) if model.end_date.nil?
+    model.end_date.to_date.to_fs(:date)
+  end
+
+  def start_date_month
+    return I18n.t(:present) if model.start_date.nil?
```

Comment:
if has no start date it did not start, so should not have this record

Reply:
that's true. solved.